tests/driver_netdev/Makefile

```diff
@@ -0,0 +1,28 @@
+APPLICATION = driver_netdev
+
+include ../Makefile.tests_common
+
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+# include specific driver makefiles in order to test compilation
+# in order to make the test application work, comment all but one
+include Makefile.enc28j60
+include Makefile.encx24j600
+include Makefile.kw2xrf
+include Makefile.xbee
+
+# test if multiple drivers are compiled in
+ifneq (x,$(DRIVERS))
```

Comment:
This always seem to be the case, when I parse the included makefiles correctly. Please explain.